projects/fusion-ui/components/daterange/common/base/daterange.base.component.ts

```diff
@@ -96,6 +97,15 @@ export abstract class DaterangeBaseComponent extends ApiBase implements OnInit,
     /** @internal */
     defaultPlaceholder = DEFAULT_PLACEHOLDER_TEXT;
 
+    /** @internal for time selector */
+    fcHasTimeSelector = new FormControl<boolean>(false);
+    /** @internal */
+    fcStartTime = new FormControl('00:00', [Validators.required]);
+    /** @internal */
+    fcEndTime = new FormControl('23:59', [Validators.required]);
```

Comment:
please check with requiermetns

Reply:
Checked with @amitmo50 - this defaults is ok.